website/docs/d/service_account.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_service_account"
+sidebar_current: "docs-vcd-datasource-service-account"
+description: |-
+  Provides a data source to read VCD Service Accounts.
+---
+
+# vcd\_service\_account
+
+Provides a data source to read VCD Service Accounts.
+
+Supported in provider *v3.10+* and VCD 10.4+.
+
+## Example Usage 1
```

Comment:
Just a nit, but given that there's only one example:
```suggestion
## Example Usage
```

Reply:
Fixed.